src/tests/python_tests.txt

```diff
@@ -351,10 +351,10 @@ test_module
 test_modulefinder
 test_msilib
 test_multibytecodec
-test_multiprocessing_fork
-test_multiprocessing_forkserver
-test_multiprocessing_main_handling
-test_multiprocessing_spawn
```

Comment:
Previously in those tests multiprocessing was correctly detected as missing at runtime and now it's no longer the case. Maybe a better solution is to monkeypatch the problematic method? In any case, it would be worth doing these multiprocessing changes in a separate PR.

Reply:
Opened a separate PR #2094.